DataFormats/Math/interface/GraphWalker.h

```diff
@@ -70,8 +70,8 @@ GraphWalker<N,E>::GraphWalker(const Graph<N,E> & g)
  : graph_(g)
 {  // complexity = (no nodes) * (no edges)
    graph_.findRoots(root_);
-   stack_.push_back(edge_range(root_.begin(),root_.end())); 
-   if (root_.size()) {
+   stack_.emplace_back(edge_range(root_.begin(),root_.end())); 
+   if (root_.empty()) {
```

Comment:
During last commit https://github.com/cms-sw/cmssw/pull/20020/commits/1e62fd6fe2504fa47becc9627d45d10308e498e0 the "!" was reverted here: please fix,

I didn't spot other examples of that kind in the DetectorDescription part of this PR, but please double check